LICENSE

```diff
@@ -1,10 +1,6 @@
-MIT License
+The MIT License (MIT)
 
-Copyright (c) 2017
-
-Christopher Pappas <damassi.pappas@gmail.com>
-Alan Johnson <alan@artsymail.com>
-Artsy <https://artsy.net>
+Copyright (c) 2018 Artsy Inc. <dev@artsymail.com>
```

Comment:
IMO you two should still be on there too

Reply:
Cool, will update 👍